openreview/journal/assignment.py

```diff
@@ -160,3 +160,117 @@ def compute_affinity_scores(self, note, committee_id):
             return response.get('results', [])
         except Exception as e:
             raise openreview.OpenReviewException('Error computing affinity scores: ' + str(e))
+
+    def setup_ae_matching(self, label):
+
+        journal = self.journal
+        submitted_submissions = self.client.get_notes(invitation= journal.get_author_submission_id(), content = { 'venueid': journal.submitted_venue_id })
+
+        for submitted_submission in tqdm(submitted_submissions):
+            ## Get AE group is empty
+            if not self.client.get_group(journal.get_action_editors_id(submitted_submission.number)).members:
+                ## Get AE recommendations
+                ae_recommendations = self.client.get_edges(invitation=journal.get_ae_recommendation_id(), head=submitted_submission.id)
+                if len(ae_recommendations) >= 3:
+                    ## Mark the papers that needs assignments. use venue: "TMLR Assigning AE" and venueid: 'TMLR/Assign_AE_20220928'
```

Comment:
What does the number in the venueid mean? Is this supposed to just be `TMLR/Assigning_AE`?

Reply:
sorry, outdated comment :( I was trying to use an unique venueid for each batch of submissions but it is too hard to do it so I decided to use the same venueid `TMLR/Assigning_AE` every time we setup the matching.